### ISSUE TITLE
Is this project still active??

### ISSUE THREAD

**Issue by Pickylightbulb** [GreenJon902/MemriseBot#2]
I tried to use it but it just keeps reloading the quiz, help?
Does it still work with the curent version of memrise? if yes can you explain me real quick how to proceed because I don't seem to  be able to make it work :(

**Comment by GreenJon902** [comment]
Yes, it should do that but go through the question page first! It should review, paste in the answer and then repeat that some times, does yours not do that?

Also I will push a newer version tomorrow as the one at the moment is a bit trash.

**Comment by Pickylightbulb** [comment]
Mine just refresh the question endlessly

I just don't get what I have to do here : 

Set the positions of the Review and the levels button by going through once, when you click change then you have 5 seconds to get the mouse over the button.


what is change?

**Comment by Pickylightbulb** [comment]
I see there is a new version now but I am still too dumb to figure out how it works, what am I suppose to put in translation?
btw I'm french so for example shave (the only word that is not ignored) is raser, so I put raser for the translation right?
and also when I click next the command prompt is spammed with `failed name 'path' is not defined`

**Comment by GreenJon902** [comment]
Yes, sorry about the program information being very wrong, I changed they system a while back and forgot to change the text, by the translation I mean this:
![Screenshot 2021-11-27 at 16 52 30](https://user-images.githubusercontent.com/37347905/143690013-ed1f83ca-3033-4e34-9244-aeece2eb9355.png)
Circled in red is what should be pasted into the translation box.

As for the
>`failed name 'path' is not defined`

That should now have a half fix of it knows when it has not get the path. I am now going to try to make this project more user friendly so that should come out pretty soon.

EDIT: This has now been released, .exe and .app versions should hopefully also be added soon!
